eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -69,6 +69,7 @@ jobs:
           CoverageArg: $(CoverageArg)
           EnvVars:
             AZURE_RUN_MODE: 'Live' #Record, Playback
+            RESOURCE_REGION: $(TestRegionOverride)
```

Comment:
After you add the parameter at the top. Swap to use `#{{ parameters.RegionOverride }}`